app/src/main/java/org/mozilla/fenix/AppRequestInterceptor.kt

```diff
@@ -71,10 +71,8 @@ class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
     private fun improveErrorType(errorType: ErrorType): ErrorType {
         // This is not an ideal solution. For context, see:
         // https://github.com/mozilla-mobile/android-components/pull/5068#issuecomment-558415367
-        val cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
-        val activeNetwork: NetworkInfo? = cm.activeNetworkInfo
-        @Suppress("DEPRECATION") // NetworkCallback is not appropriate for this use case
-        val isConnected: Boolean = activeNetwork?.isConnectedOrConnecting == true
+
+        val isConnected: Boolean = context.connectivityManager.isOnline()
```

Comment:
I prefer to use Android KTX over Anko
```suggestion
        val isConnected: Boolean = context.getSystemService<ConnectivityManager>!!.isOnline()
```

Reply:
I did see an issue mentioning "consider moving away from anko", was not sure.